p2p/swarm.go

```diff
@@ -395,7 +397,7 @@ func (s *swarm) retryOrReplace(key p2pcrypto.PublicKey) {
 
 // periodically checks that our clock is sync
 func (s *swarm) checkTimeDrifts() {
-	checkTimeSync := time.NewTicker(config.TimeConfigValues.RefreshNtpInterval)
+	checkTimeSync := time.NewTicker(timeConfig.Values.RefreshNtpInterval)
```

Comment:
this also uses `Values` but values aren't loaded from anywhere.

Reply:
Sorry, but I'm not quite understand the problem. Do you mean we should use the config loaded in `root.go`?

Reply:
what i'm saying is, that if we keep using `Values` (which is what we're probably going to do since the `timesync` package is kinda "static" right now). we should atleast make sure the values from the config file and cli flags are loaded into it.

Reply:
So we should use the config in `cmd` package, right?

Reply:
For the `TimeConfig` Values we'll have to use a global var like we're using now (`Values`). the thing it we set it do `DefaultConfig()`. we need to set the values we retrieved from inputs in `main.go` to also affect the timeconfig stored in `Values`. 
Other than that I prefer we'd use the config passed down from `main` anywhere else.

Reply:
Passing down from `main` is better. Please check this commit 6aeb39df2162600f66b397b0b57f2171ca1857a2.